### ISSUE TITLE
Apache config parse failure with missing double quote in directive argument

### ISSUE THREAD

**Issue by domcleal** [letsencrypt/letsencrypt#1724]
Originally reported by @hokascha at https://github.com/letsencrypt/letsencrypt/issues/1294#issuecomment-161916875, refiling as a new issue.

The SSLCipherSuite directive here is missing an ending double quote, but Apache appears to parse it fine (it passes a config test anyway) while Augeas (under LE) fails.

<pre>
	SSLProtocol all -SSLv2 -SSLv3
        SSLCipherSuite "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRS$
        SSLCertificateFile /etc/ssl/certs/example.com.crt
</pre>

Full config at https://gist.github.com/hokascha/87da802584fa2b0675cc.

Waiting on a fix in Augeas, filed as https://github.com/hercules-team/augeas/issues/330.

**Comment by hokascha** [comment]
Adding the missing ending double quote fixes the problem. Even better, adding the missing CipherSuites fixes it too ;-) Looks like I accidentially missed to copy the full line from my template config on that host. Full line should be (and parses ok):
```
SSLCipherSuite "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS"
```

**Comment by domcleal** [comment]
Ah yes, I see what you mean - the `$` gives it away now!  As it happens, we came across another example of this at the end of #1531, where it's valid syntax for ErrorDocument messages etc.
